_data/projects/xander.yml

```diff
@@ -0,0 +1,11 @@
+name: Xander
+desc: A Nim web application framework
+site: https://github.com/sunjohanday/xander
+tags:
+- nim
+- web
+- framework
+- library
+upforgrabs:
+  name: Help needed to keep developing the project
```

Comment:
When this PR gets merged and listed, this entry should refer to the label name used to mark issues that are available for new contributors to get involved with. Is this something you're interested in setting up?